### PR TITLE
Adding a bit more of clarfication on default channel configs

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

## Proposed Changes 

- adding a explicit section for setting some specific channel implementation for the default broker (the one that's there OOTB in eventing)
- and changing the section for the default channel to be more explict about options

### CODE REVIEW

**Comment by lberk** [comment]
@abrennan89 looks good from a technical perspective, would you be willing to stamp an approval on this?

**Comment by chaodaiG** [comment]
Just for testing OWNERS file
/approve

File: docs/install/operator/configuring-eventing-cr.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ All Knative Eventing ConfigMaps are created in the same namespace as the Knative
 Knative Eventing has multiple ConfigMaps that are named with the prefix `config-`.
 The `spec.config` in the KnativeEventing CR has one `<name>` entry for each ConfigMap, named `config-<name>`, with a value which will be used for the ConfigMap `data`.
 
-### Setting the default channel for the broker
+### Setting a default channel
 
-If you are using a channel-based broker, you can change the default channel type for the broker from InMemoryChannel to KafkaChannel, by updating the `config-br-default-channel` ConfigMap. You can do this by modifying the KnativeEventing CR:
+If you are using different channel implementations, like the KafkaChannel, or you want a specific configuration of the InMemoryChannel to be default you can change the default behavior by updating the `default-ch-webhook` ConfigMap. You can do this by modifying the KnativeEventing CR:
```
Comment (omerbensaadon) [inline docs/install/operator/configuring-eventing-cr.md:51]:
```suggestion
If you are using different channel implementations, like the KafkaChannel, or you want a specific configuration of the InMemoryChannel to be the default configuration, you can change the default behavior by updating the `default-ch-webhook` ConfigMap. 

You can do this by modifying the KnativeEventing CR:
```
Separating the actions from the context

File: docs/install/operator/configuring-eventing-cr.md [hunk 2]
```diff
@@ -46,9 +46,9 @@ All Knative Eventing ConfigMaps are created in the same namespace as the Knative
...
```
Comment (omerbensaadon) [inline docs/install/operator/configuring-eventing-cr.md:52]:
```suggestion
```

File: docs/install/operator/configuring-eventing-cr.md [hunk 3]
```diff
@@ -79,6 +79,27 @@ spec:
 
 **NOTE:** The `clusterDefault` setting determines the global, cluster-wide default channel type. You can configure channel defaults for individual namespaces by using the `namespaceDefaults` setting.
 
+### Setting the default channel for the broker
+
+If you are using a channel-based broker, you can change the default channel type for the broker from InMemoryChannel to KafkaChannel, by updating the `config-br-default-channel` ConfigMap. You can do this by modifying the KnativeEventing CR:
```
Comment (omerbensaadon) [inline docs/install/operator/configuring-eventing-cr.md:84]:
```suggestion
If you are using a channel-based broker, you can change the default channel type for the broker from InMemoryChannel to KafkaChannel, by updating the `config-br-default-channel` ConfigMap. 

You can do this by modifying the KnativeEventing CR:
```

File: docs/install/operator/configuring-eventing-cr.md [hunk 4]
```diff
@@ -79,6 +79,27 @@ spec:
...
+
```
Comment (omerbensaadon) [inline docs/install/operator/configuring-eventing-cr.md:85]:
```suggestion
```

**Review by omerbensaadon — COMMENTED** [review]
Minor tweaks, but otherwise lgtm!

File: docs/install/operator/configuring-eventing-cr.md [hunk 5]
```diff
@@ -46,10 +46,11 @@ All Knative Eventing ConfigMaps are created in the same namespace as the Knative
...
 The `spec.config` in the KnativeEventing CR has one `<name>` entry for each ConfigMap, named `config-<name>`, with a value which will be used for the ConfigMap `data`.
 
-### Setting the default channel for the broker
+### Setting a default channel
 
-If you are using a channel-based broker, you can change the default channel type for the broker from InMemoryChannel to KafkaChannel, by updating the `config-br-default-channel` ConfigMap. You can do this by modifying the KnativeEventing CR:
+If you are using different channel implementations, like the KafkaChannel, or you want a specific configuration of the InMemoryChannel to be the default configuration, you can change the default behavior by updating the `default-ch-webhook` ConfigMap. 
 
+You can do this by modifying the KnativeEventing CR:
 ```yaml
```
Comment (abrennan89) [inline docs/install/operator/configuring-eventing-cr.md:54]:
```suggestion

```yaml
```

File: docs/install/operator/configuring-eventing-cr.md [hunk 6]
```diff
@@ -79,6 +81,28 @@ spec:
 
 **NOTE:** The `clusterDefault` setting determines the global, cluster-wide default channel type. You can configure channel defaults for individual namespaces by using the `namespaceDefaults` setting.
 
+### Setting the default channel for the broker
+
+If you are using a channel-based broker, you can change the default channel type for the broker from InMemoryChannel to KafkaChannel, by updating the `config-br-default-channel` ConfigMap. 
+
+You can do this by modifying the KnativeEventing CR:
+```yaml
```
Comment (abrennan89) [inline docs/install/operator/configuring-eventing-cr.md:89]:
```suggestion

```yaml
```

**Pull request merged**